src/test/java/uk/gov/hmcts/reform/slc/services/steps/getpdf/PdfCreatorTest.java

```diff
@@ -34,14 +38,27 @@ public void should_require_letter_to_not_be_null() {
     }
 
     @Test
-    public void should_return_pdf_object() {
+    public void should_return_list_of_pdfs() {
         given(client.generateFromHtml(any(), any()))
             .willReturn("hello".getBytes());
 
-        PdfDoc pdfDoc = pdfCreator.create(new Letter("template", emptyMap(), "type", "service"));
+        List<PdfDoc> pdfs = pdfCreator.create(
+            new Letter(
+                asList(
+                    new Document("t1", emptyMap()),
+                    new Document("t2", emptyMap())
+                ),
+                "type",
+                "service"
+            )
+        );
+
+        assertThat(pdfs).hasSize(2);
+
+        assertThat(pdfs.get(0).content).isEqualTo("hello".getBytes());
+        assertThat(pdfs.get(0).filename).isNotEmpty();
```

Comment:
Not strictly related with this pull request, but we're missing tests checking if the file name is built correctly.

Reply:
there is a separte suite of tests for file name generation

Reply:
👍 missed this